src/main/java/frc/robot/commands/OpenIntake.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.commands.smartmotorcontrollergenericsubsystem.MoveSmartMotorControllerGenericSubsystem;
+import com.spikes2212.util.UnifiedControlMode;
+import frc.robot.subsystems.IntakePlacer;
+
+import java.util.function.Supplier;
+
+public class OpenIntake extends MoveSmartMotorControllerGenericSubsystem {
```

Comment:
add isFinished that also makes the command finish when the intake is down (See how the subsystem is made)